packages/design-system/.storybook/preview.tsx

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2025 Hypergiant Galactic Systems Inc. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+// biome-ignore lint/correctness/noUnusedImports: fix later
+import React from 'react';
+import type { Preview } from '@storybook/react';
+import { withActions } from '@storybook/addon-actions/decorator';
+import { ThemeProviderDecorator } from './components';
+
+const preview: Preview = {
+  parameters: {
+    controls: {
+      matchers: {
+        color: /(background|color)$/i,
+        date: /Date$/i,
+      },
+    },
+    backgrounds: {
+      default: 'dark',
+      values: [{ name: 'dark', value: 'rgb(11,11,11)' }],
+    },
+    layout: 'centered',
+    actions: { argTypesRegex: '^on.*' },
```

Comment:
There's going to be a potential issue with automatically binding to all events. There's one (need to find the comment in our code) in RAC that changes the behavior of the component

Reply:
this is a remnant of some experimentation I was doing and isn't actually necessary. we're explicitly specifying actions to capture in the stories. i will remove this.

Reply:
cleaned up here: https://github.com/gohypergiant/standard-toolkit/pull/167/commits/0789ff6d84c8a8f1508b6b01bc9efd609fa4224d